provider/aws.go

```diff
@@ -391,7 +391,7 @@ func newChange(action string, endpoint *endpoint.Endpoint) *route53.Change {
 		}
 		change.ResourceRecordSet.ResourceRecords = []*route53.ResourceRecord{
 			{
-				Value: aws.String(endpoint.Target),
+				Value: aws.String(endpoint.Targets[0]),
```

Comment:
To make multi-target really work this should actually create a `ResourceRecord` for each `Targets[i]` I believe. I agree to not have this in-scope for this PR. Just want to raise awareness.